src/main/java/me/hydos/rosella/util/HashUtil.java

```diff
@@ -6,7 +6,14 @@
 
 public class HashUtil {
 
+    private static final boolean USE_XXHASH_WITH_SEED = Boolean.parseBoolean(System.getProperty("rosella:xxhash_with_seed"));
+    private static final int SEED = 0xC7AB;
```

Comment:
this should be a property as well

Reply:
Why so? This is a workaround for a bug anyway